gmail/types.bal

```diff
@@ -68,17 +68,17 @@ public type MessageRequest record {
     @display {label: "Recipient"}
     string recipient;
     @display {label: "Subject"}
-    string subject?;
-    @display {label: "Message Body"}
-    string messageBody;
+    string subject;
     @display {label: "Content Type"}
     string contentType;
-    @display {label: "Sender"}
-    string sender;
+    @display {label: "Message Body"}
```

Comment:
Oder should be like `Recipient`, `Subject` then `Message Body`. Content type should come after that. BTW, is it a mandatory field?

Reply:
Ack. Google API it is not mandatory, Since we make the messagebody as mandatory here , we made  Content type also a mandatory to organize the body with attachments when they have. But If we want we can make as optional